airflow/config_templates/config.yml

```
@@ -2591,3 +2591,23 @@ sensors:
       type: float
       example: ~
       default: "604800"
+scarf_analytics:
```

The property is very much directly referring to scarf. I know we decided to use this (at the moment) but it might be that we change in future. Therefore I'd propose not to bind the config entry to the backend service/product name. I propose to rename it with a neutral scope, e.g.:
```suggestion
analytics_collection:
```
+ of course other references in code need to be adjusted.

---

Updated